04 Strategy Library/999 Seasonality and Session Effect in Index Futures/03 Method.html

```diff
@@ -0,0 +1,124 @@
+<h3>Step 1:  Initialization</h3>
+
+<p>
+	We set up an 8-year long backtest period from 2008-01-01 to 2015-12-31. Then, we choose the E-mini S&P 500 futures, sort them in ascending order according to days to expiration and pick the nearest expiration contract. After that, we set the Bullish Season and the Bearish Season based on months and set the Night Session based on time of a day. We trigger the open and the close of trades with self.Schedule.On() function.
+</p>
+
+<div class="section-example-container">
+
+<pre class="python">
+	self.SetStartDate(2008, 1, 1)  # Set Start Date
```

Comment:
Use [this HTML renderer](https://htmledit.squarefree.com/) to double check that all of the code indentations will appear properly.